batch/batch/stoppable_server.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
```

Comment:
Should this import gear and use the logging config from there?

Reply:
This class assumes that you've already configured logging. I don't know if you can configure logging twice?